src/scripts/components/dateselector/wv.dateselector.js

```diff
@@ -99,6 +107,46 @@ class dateSelector extends React.Component {
           minDate={this.props.minDate}
           blur={this.blur.bind(this)}
         />
+        {this.state.maxZoom >= 4 &&
+          <DateInputColumn
+            step={1}
+            startDate={new Date(2000)}
+            today={new Date()}
+            date={this.state.date}
+            type="hour"
+            height={this.props.height}
+            width={this.props.width}
+            updateDate={this.updateDate.bind(this)}
+            value={util.pad(this.state.date.getUTCHours(), 2, '0')}
+            tabIndex={4}
+            focused={(this.state.tab === 4)}
+            nextTab={this.nextTab.bind(this)}
+            maxDate={this.props.maxDate}
+            minDate={this.props.minDate}
+            blur={this.blur.bind(this)}
+          />
+        }
+        {this.state.maxZoom >= 4 && <div className="input-time-divider">:</div> }
```

Comment:
In this case you could also pull this out into it's own function so it only has to run the logic `this.state.maxZoom >= 4` once. 

That would look something like:
```
  renderSubdaily() {
    if(this.state.maxZoom >= 4) {
      return ( 
        <div>
          <div className="input-time-divider">:</div>
          <DateInputColumn
            step={10}
            startDate={new Date(2000)}
            today={new Date()}
            date={this.state.date}
            type="minute"
            height={this.props.height}
            width={this.props.width}
            updateDate={this.updateDate.bind(this)}
            value={util.pad(this.state.date.getUTCMinutes(), 2, '0')}
            tabIndex={5}
            focused={(this.state.tab === 5)}
            nextTab={this.nextTab.bind(this)}
            maxDate={this.props.maxDate}
            minDate={this.props.minDate}
            blur={this.blur.bind(this)}
          />
          <div className="input-time-zmark">Z</div>
        </div>
      );
    }
  }
```

Both implementations work. Feel free to keep what you have or implement this. Either way.

Reply:
The initial reason I didn't do this is that react's wrapping div was affecting the tabbing. I have updated to react 16.2.0 which introduces fragments. So, I will give this a go using fragments.

Reply:
I have added a `renderSubdaily()` method using `React.Fragment`. Unfortunately `eslint` has no update to handle the short `<>`, `</>` syntax so I had to write the long-hand `<React.Fragment>`, `</React.Fragment>`